### DISCUSSION TITLE
Different permission for each panel on insights

### DISCUSSION THREAD

**Discussion by YasminCastro** [directus/directus#13419] [Feature Requests]
I would like to each role see a different panel in insights. 
I have several different roles, and i wanted that each one of them see a especific panel. example: The role 1 see only panel 1, role 2 see only panel 2... now i can get the role to only see everything or to see nothing.

![Snipaste_2022-05-19_14-39-00](https://user-images.githubusercontent.com/62770317/169363265-449f12f1-0af0-4b31-9e67-0943b5862514.png)

**Comment by azrikahar** [comment]
You should be able to configure custom read permissions to Directus Panels for the roles so that they can only view certain panels (or use negative operators to do the inverse):

https://user-images.githubusercontent.com/42867097/169957848-503c5538-24dd-4143-b415-7d8ecd74dbbd.mp4

(The dashboard ID was added in case we are using panel name instead of ID, as there maybe panels with the same name on different dashboards, but feel free to omit it if you're already using ID)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
